### ISSUE TITLE
downloadLicenses fails trying to expand a pom as a Zip

### ISSUE THREAD

**Issue by robertmk** [hierynomus/license-gradle-plugin#90]
Hi,

I tried to use the plugin to show a report about the used licenses but got this error:

./gradlew downloadLicenses
:downloadLicenses FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':downloadLicenses'.
> Could not expand ZIP '/Users/krueger/.gradle/caches/modules-2/files-2.1/de.lesspain/de.lesspain.jni-darwin_x86_64/0.14-SNAPSHOT/61c61a5e60850e8deb81247ed3ae1566bc0c38c8/de.lesspain.jni-darwin_x86_64-0.14-SNAPSHOT.pom'.

Any idea what might be causing this?

**Comment by hierynomus** [comment]
What does your `build.gradle` look like?

**Comment by robertmk** [comment]
It is a huge project and in a closed environment, so unfortunately I cannot send you the buildfile itself. Anything in particular I can check or any idea you got, why it thinks the pom is a zip?

**Comment by hierynomus** [comment]
Is one of the dependencies maybe defined as `@pom`? So that only the pom
file is downloaded?

2015-10-27 15:18 GMT+01:00 robertmk <notifications@github.com>:

**Comment by hierynomus** [comment]
If you've got some more info, or are able to isolate the cause, I would love a reproducible testcase :)

**Comment by robertmk** [comment]
No, I am not aware of anything like that. The particular file where this
happens is a jar file, which is only special in that it contains native
libraries for use with JNI.

On Tue, Oct 27, 2015 at 3:19 PM, Jeroen van Erp <notifications@github.com>
wrote:

> Is one of the dependencies maybe defined as `@pom`? So that only the pom
> file is downloaded?
>
> 2015-10-27 15:18 GMT+01:00 robertmk <notifications@github.com>:
>
> > It is a huge project and in a closed environment, so unfortunately I
> > cannot send you the buildfile itself. Anything in particular I can check
> or
> > any idea you got, why it thinks the pom is a zip?
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/hierynomus/license-gradle-plugin/issues/90#issuecomment-151510239
> >
> > .
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hierynomus/license-gradle-plugin/issues/90#issuecomment-151510626>
> .
>



-- 
Robert Krüger
Managing Partner
Lesspain GmbH & Co. KG

www.lesspain-software.com

**Comment by robertmk** [comment]
I'll try but at the moment I have no idea how to do that.

On Tue, Oct 27, 2015 at 10:41 PM, Jeroen van Erp <notifications@github.com>
wrote:

> If you've got some more info, or are able to isolate the cause, I would
> love a reproducible testcase :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hierynomus/license-gradle-plugin/issues/90#issuecomment-151653985>
> .
>



-- 
Robert Krüger
Managing Partner
Lesspain GmbH & Co. KG

www.lesspain-software.com
